### PR TITLE
Changes for Thunderbird 79+

### PR DESCRIPTION
**Pull request by jpyllman**
Some changes that is needed for versions 79+ of thunderbird.

### CODE REVIEW

**Comment by jpyllman** [comment]
Probably need more changes, look at https://developer.thunderbird.net/add-ons/updating/tb91/changes

**Comment by tessarakt** [comment]
> Probably need more changes, look at https://developer.thunderbird.net/add-ons/updating/tb91/changes

Might be, but for TB 90.0b1 it works fine. Thanks a lot!

**Comment by jpyllman** [comment]
this is a PR for issue #50

File: src/content/quickmove.js [hunk 1]
```diff
@@ -266,9 +266,8 @@ var quickmove = (function() {
         allNames.push(aFolder.prettyName.toLowerCase());
 
         if (aFolder.hasSubFolders) {
-          let myenum = aFolder.subFolders;
-          while (myenum.hasMoreElements()) {
-            processFolder(myenum.getNext().QueryInterface(Ci.nsIMsgFolder), excludeArchives);
+          for(let xFolder of aFolder.subFolders) {
```
Comment (kewisch) [inline src/content/quickmove.js:269]:
```suggestion
          for (let xFolder of aFolder.subFolders) {
```

**Pull request merged**